tests/roundtrip.rs

```diff
@@ -0,0 +1,125 @@
+#[cfg(test)]
+mod tests {
+    use factorio_blueprint::roundtrip_blueprint_test;
+
+    #[test]
+    fn test_roundtrips_empty() {
+        // Empty blueprint book
+        roundtrip_blueprint_test("0eNqrVkrKKU0tKMrMK4lPys/PVrKqVsosSc1VskJI6IIldJQSk0syy1LjM/NSUiuUrAx0lMpSi4oz8/OUrIwsDE3MLY3MzQyNTUwMDGprAVU7HPM=");
+        // Empty blueprint (in a blueprint book, cannot export otherwise)
+        roundtrip_blueprint_test("0eNqrVkrKKU0tKMrMK4lPys/PVrKqRogUK1lFI3FBcpklqblKVkhiOkplqUXFmfl5SlZGFoYm5pZG5maGxiYmBga1OkqZeSmpFUpWBrWxOhg6dcHW6SglJpdklqXGw5TiMa8WAEFMONI=");
+        // Empty deconstruction planner
+        roundtrip_blueprint_test("0eNpdy0EKgCAQAMC/7NlATbL8TIQtIdgaunYR/16XLl0HpsGOPlHhXD2HROsVNyLM4BoUZA50FHBUYxQQGE9wvzB8QcCNubwCTs/K2EXbSY3GSNn7A7BrI8Q=");
+        // Empty upgrade planner
+        roundtrip_blueprint_test("0eNo1ykEKgCAQBdC7/LWBmmR5mRAaRLBJ1NqId2/lW7+ON4fiLzpz8sxU4DoqtRY5VDh+UxKIjW64OZc5BT4qNT4Mp3dl7KHtplZjpBzjB4YmHaw=");
```

Comment:
These examples are useful to have, but why embed them directly here instead of using the `tests/examples` folder?

If the argument is that tests are more informative if they can fail on an individual basis, then we should import or write a macro which creates a test per example file in the examples folder.

Reply:
Yea, it would make more sense to move these to tests/examples now. Wrote them originally before replacing the roundtrip functionality in the other file, with some basic comments to go along. Should be able to get descriptive enough in the filenames.